### ISSUE TITLE
Redo colours for the site, general improvements for light mode

### ISSUE THREAD

**Issue by banner4422** [thebentobot/bento-the-website#217]
Adam was nice to share a tool for colour contrast checking, which I really lack in some places, especially for the light mode. I feel like the gradient on the frontpage could be better too.

Double checking colours with https://webaim.org/resources/contrastchecker/ and redo places like gradient etc.
